packages/date-time-utilities/config/tests.js

```diff
@@ -0,0 +1,10 @@
+/** Jest test setup file. */
```

Comment:
Remove this since this package won't likely have tests requiring Enzyme/React.

Reply:
Removed